packages/sqs-receiver-service/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "sqs-receiver-service",
+  "version": "1.0.0",
+  "description": "SQS Receiver service",
+  "main": "index.js",
+  "type": "module",
+  "private": true,
+  "license": "SEE LICENSE IN LICENSE",
+  "engines": {
+    "node": ">=13"
+  },
+  "scripts": {
+    "run": "pm2 start ecosystem.config.yml",
+    "test": "jest"
+  },
+  "dependencies": {
+    "@hapi/joi": "^17.1.0",
+    "aws-sdk": "^2.627.0",
+    "babel-jest": "^25.1.0",
```

Comment:
babel-jest  is not required as a dependency (should be a dev dependency but this is already present in the top-level package.json)

Reply:
done